openhands/resolver/send_pull_request.py

```diff
@@ -24,6 +25,82 @@
 from openhands.utils.async_utils import GENERAL_TIMEOUT, call_async_from_sync
 
 
+def get_issue_handler(
```

Comment:
The addition of this function should not be necessary given that we are just adding bit bucket support.